### ISSUE TITLE
it is possible to set a cage file path?

### ISSUE THREAD

**Issue by Lluc3D** [orangeduck/Python-xNormal#2]
I see the option to activate the "cage" true or false ( ("use_cage", "UseCage", False), ) but I can't see where I have to set the path for the cage file.

**Comment by orangeduck** [comment]
Hi Lluc - I might just need to update this project for the newest version of xNormal since I've not updated it in a while but you can try just passing in a string of the cage file path for that option and seeing what happens.

**Comment by Lluc3D** [comment]
I'm planing to use your project for a small addon for Blender, I have been testing it and still works quite well. Thanks for the help.

**Comment by orangeduck** [comment]
Great! No problem 👍

**Comment by moppius** [comment]
I have added support for this in PR #3

**Comment by moppius** [comment]
This issue can be closed now, since a4f005220d31d1e9085a7cbcc1ef46e70cff2753 has been merged.

You can set a cage file path per lowpoly by using the following syntax:
```
low_mesh_path = "C:\\Model\\MyLowMesh.obj"
cage_mesh_path = "C:\\Model\\MyLowMesh_Cage.obj"
xNormal.low_mesh_options(low_mesh_path, use_cage=True, cage_file=cage_mesh_path)
```
Bear in mind that the Cage mesh topology must be identical to the Low mesh it's associated with.

**Comment by orangeduck** [comment]
Thanks!
